VERSION

```diff
@@ -1 +1 @@
-3.14.0
\ No newline at end of file
+3.16.0
```

Comment:
CI Tests failed because v3.16.0 in upstream(OPA) does not exist yet. I guess

Reply:
Oh, true. I didn't think about that. I've disabled fast-forwarding for now. I'll set this to 3.15 and we can bump it and then re-enable fast-forwarding when we want to release the operator. That probably makes more sense than trying to guess which version we're going to release.